project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -739,16 +738,25 @@ function c4m_og_open_group_access_activate_pending_members($node) {
   foreach (og_membership_load_multiple($ids) as $membership) {
     $membership->state = OG_STATE_ACTIVE;
     $membership->save();
+
+    // Send activation message to member.
+    $variables = array();
+    $account = user_load($membership->etid);
+    $message = message_create('c4m_member_welcome_message', $variables, $account);
+
+    $wrapper = entity_metadata_wrapper('message', $message);
+   // $wrapper->save();
+    message_notify_send_message($wrapper->value(), array('mail' => $account->mail));
```

Comment:
I don't think you need to pass `array('mail' => $account->mail)` -- it will already be taken from the `$message->uid`